### PR TITLE
Alnylam

### PR DESCRIPTION
**Pull request by laudickson**
ya boi just got hired back into biotech lol

### CODE REVIEW

File: views/home/amalgam.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-#past.page
+#amalgam.page
   .text-black.right-align-text
     %p
-      Prior to LevelUp, I had a brief stint at
+      Many, many summers ago, I had a brief stint at
       %a{href: "http://amalgam.co", target: "_blank", rel: "noreferrer"}Amalgam
-      where I did a bunch of front-end stuff.
+      building websites from concepts.
```
Comment (jscho13) [inline views/home/amalgam.haml:6]: Mhmm websites from concept very sophisticated I like

**Review by jscho13 — APPROVED** [review]

File: views/home/amalgam.haml (hunk 1 above)
Comment (laudickson) [reply views/home/amalgam.haml:6]: 😎

**Comment by laudickson** [comment]
it is good

File: public/stylesheets/sass/pages/_alnylam.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    animation-iteration-count: infinite;
+    animation-name: var(--name);
+    animation-timing-function: var(--tf);
+    display: inline-block;
+    position: relative;
+    transform-origin: 50% 100%;
+    z-index: 1;
+  }
+
+  .twirl {
```
Comment (jscho13) [inline public/stylesheets/sass/pages/_alnylam.scss:17]: holy shit

Comment (laudickson) [reply public/stylesheets/sass/pages/_alnylam.scss:17]: there exists a js library that auto splits words into individual characters in their own span with '.char' class. v useful

https://splitting.js.org/
https://github.com/laudickson/portfolio/pull/19/files#diff-520e8d912e4640d2a35edc95284d9e7e35c31a5ed6684872ccd4f946184f2946R10

**Pull request merged**